File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/NuroOUT0101U02GetType.java
package nta.med.data.model.ihis.nuro;

import java.io.Serializable;

public class NuroOUT0101U02GetType implements Serializable {
	 private String type;
	 private String typeName;
	public NuroOUT0101U02GetType(String type, String typeName) {
		super();
		this.type = type;
		this.typeName = typeName;
	}
	public String getType() {
		return type;
	}
	public void setType(String type) {
		this.type = type;
	}
	public String getTypeName() {
		return typeName;
	}
	public void setTypeName(String typeName) {
		this.typeName = typeName;
	}
	 
	 
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/NuroOUT0101U02GetType.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable ]
	public class NuroOUT0101U02GetType
	{
		private String _type;
		private String _typeName;

		public String Type
		{
			get { return this._type; }
			set { this._type = value; }
		}

		public String TypeName
		{
			get { return this._typeName; }
			set { this._typeName = value; }
		}

		public NuroOUT0101U02GetType() { }

		public NuroOUT0101U02GetType(String type, String typeName)
		{
			this._type = type;
			this._typeName = typeName;
		}

	}
}